ivy/data_classes/container/general.py

```diff
@@ -4060,6 +4060,114 @@ def array_equal(
             map_sequences=map_sequences,
         )
 
+    @staticmethod
+    def _static_copy_array(
+        dst: Union[ivy.Array, ivy.Container],
+        src: Union[ivy.Array, ivy.Container],
+        /,
+        *,
+        key_chains: Optional[Union[List[str], Dict[str, str], ivy.Container]] = None,
+        to_apply: Union[bool, ivy.Container] = True,
+        prune_unapplied: Union[bool, ivy.Container] = False,
+        map_sequences: Union[bool, ivy.Container] = False,
+    ) -> None:
+        """
```

Comment:
Looks good, can you explain why this static method is required?

Reply:
In general, to copy data between containers on the class itself without needing to create instances. and I saw some of the similar PRs adding the static functions, so I gave it a go. 
Why I added it is because we can call the copy operation without needing an instance of the container class, which can be useful in some scenarios.